AirSkyBoat/AirSkyBoat
# Add Black Sallet and Plastron Synthesis Recipes

**_I affirm:_**


## Please enter a player-facing description

Enable synthesis of Black Sallet and Plastron

## What does this pull request do? (Please be technical)

Closes #3236 

Updated `synth_recipes.sql`, uncommenting the existing lines for Black Sallet and Plastron, and changing the GSM to more accurately reflect our current understandings of what the requirements were during the 75 cap era. 

Based on synthesis skill caps at the end of COP Era, neither of these synths will be able to be T1'ed, requiring 96 and 97 Blacksmithing. 

The only documented English reference to the GSM level requirement for Plastron is a comment on [FFXI Wiki](https://ffxiclopedia.fandom.com/wiki/Talk:Plastron#:~:text=Gained%20goldsmithing%20skill%20at%20level%2052.) that mentions receiving skillup at 52 GSM. If you dig into the JP Wiki, you can find a list of synths that [includes Plastron](https://ff11.jis.click/synthetic-resipe.php?t=1#:~:text=%E3%83%97%E3%83%A9%E3%82%B9%E3%83%88%E3%83%AD%E3%83%B3,%E9%9D%A9%E7%B4%B0%E5%B7%A539), that sets the GSM and Leather requirements to 59 and 39 respectively. This would explain the skillup the comment on the English FFXI Wiki.

On the same page referenced above, the [Black Sallet recipe](https://ff11.jis.click/synthetic-resipe.php?t=1#:~:text=%E3%83%96%E3%83%A9%E3%83%83%E3%82%AF%E3%82%B5%E3%83%AA%E3%83%83%E3%83%88,96%0A%E5%BD%AB%E9%87%9155) says it requires 96 Smithing and 55 Goldsmithing.   

## Steps to test these changes
1. Run the `synth_recipes.sql` file.
2. Set your account to have 100 Smithing, 59 Goldsmithing and 39 Leather.
3. Add the required synth materials to your inventory and attempt to craft Plastron and Black Sallet.
4. The materials should be accepted as a valid synth, and you should be able to craft without skillups.

## Special Deployment Considerations

Mentioned above, you will need to run the `synth_recipes.sql` file
